pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.black]
+line-length = 88
```

Comment:
We have line length rule in pylintrc set to [120](https://github.com/transferwise/pipelinewise-singer-python/blob/24bbeee261e179062d78013fb659106b26953a44/pylintrc#L286) and we're happy with it. 
Can you remove this or make it 120 too?

Reply:
done!